src/content/en/2019/javascript.md

```diff
@@ -289,24 +289,24 @@ Other top used JavaScript libraries include jQuery variants (jQuery migrate, jQu
 
 ### Frameworks and UI libraries
 
+<aside class="note">As mentioned in our <a href="./methodology#wappalyzer">methodology</a>, the third-party detection library used in HTTP Archive (Wappalyzer) has a number of limitations with regards to how it detects certain tools. For JavaScript libraries and frameworks, the limitations are described in more detail <a href="https://github.com/AliasIO/wappalyzer/issues/2450">here.</a></aside>
```

Comment:
Nit: Can we change links to Markdown?

Also can we have a better link than "here" (plus full stop was in wrong place).

What about something like this?:

```suggestion
<aside class="note">As mentioned in our (methodology)[./methodology#wappalyzer"], the third-party detection library used in HTTP Archive (Wappalyzer) has a number of limitations with regards to how it detects certain tools. There is an open issue [to improve detection of JavaScript libraries and frameworks](https://github.com/AliasIO/wappalyzer/issues/2450), which will have impacted the results presented here.</aside>
```